### PR TITLE
Adding install instructions for vs2019, windows terminal (preview), intellij ide 2019 in readme.md

### PR DESCRIPTION
**Pull request by tklecka**
## Summary of the Pull Request
Adding install instructions for vs2019, windows terminal (preview), intellij ide 2019 in README.md
 
## References
/

## PR Checklist
* [ ] Closes #xxx
* [x] CLA signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/Terminal) and sign the CLA
* [ ] Requires FONTLOG.txt to be updated
* [ ] Requires [/images/cascadia-code.png](/microsoft/cascadia-code/blob/master/images/cascadia-code.png) and/or [/images/cascadia-code-characters.png](/microsoft/cascadia-code/blob/master/images/cascadia-code-characters.png) to be updated
* [ ] I've discussed this with core contributors already. If not checked, I'm ready to accept this work might be rejected in favor of a different grand plan. Issue number where discussion took place: #xxx

## Detailed Description of the Pull Request / Additional comments
Adding install instructions for vs2019, windows terminal (preview), intellij ide 2019 in README.md with Pictures

## Before (if applicable) and After Images of the Character(s)
/

## Validation Steps Performed
/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,6 +94,40 @@ This is the planned roadmap for Cascadia Code. Please be aware that the delivery
 
 ![VS Code Settings](images/vscode-ligature-settings.png "VS Code Ligatures Setting")
 
+# Installing Cascadia Code in Visual Studio 2019
```
Comment (cinnamon-msft) [inline README.md]:
```suggestion
# Setting Cascadia Code in Visual Studio 2019
```

File: README.md [hunk 2]
```diff
@@ -94,6 +94,40 @@ This is the planned roadmap for Cascadia Code. Please be aware that the delivery
...
+2. Enter "Fonts and Colors" in search field or go to `Environment > Fonts and Colors`.
+3. Select `Text Editor` in `Show settings for:`.
+4. In the `Fonts` Dropdown select `Cascadia Code`.
+5. Press `Ok` and you're good to go.
+
+> Note: If you've installed font and it does not get applied in Visual Studio 2019, try restarting Visual Studio 2019.
+
+![Visual Studio 2019 Settings](images/vs2019-font-settings.png "Visual Studio 2019 Font Settings")  
+
+# Installing Cascadia Code in Windows Terminal (Preview)
```
Comment (cinnamon-msft) [inline README.md]:
```suggestion
# Setting Cascadia Code in Windows Terminal (Preview)
```

File: README.md [hunk 3]
```diff
@@ -94,6 +94,40 @@ This is the planned roadmap for Cascadia Code. Please be aware that the delivery
 
 ![VS Code Settings](images/vscode-ligature-settings.png "VS Code Ligatures Setting")
 
+# Installing Cascadia Code in Visual Studio 2019
+
+1. Go to `Tools > Options`in Visual Studio 2019.
```
Comment (ItzLevvie) [inline README.md]:
```suggestion
1. Go to `Tools > Options` in Visual Studio 2019.
```

File: README.md [hunk 4]
```diff
@@ -94,6 +94,40 @@ This is the planned roadmap for Cascadia Code. Please be aware that the delivery
...
 
+# Installing Cascadia Code in Visual Studio 2019
+
+1. Go to `Tools > Options`in Visual Studio 2019.
+2. Enter "Fonts and Colors" in search field or go to `Environment > Fonts and Colors`.
+3. Select `Text Editor` in `Show settings for:`.
+4. In the `Fonts` Dropdown select `Cascadia Code`.
+5. Press `Ok` and you're good to go.
+
+> Note: If you've installed font and it does not get applied in Visual Studio 2019, try restarting Visual Studio 2019.
```
Comment (ItzLevvie) [inline README.md]:
```suggestion
> Note: If you've installed the font and it does not get applied in Visual Studio 2019, try restarting Visual Studio 2019.
```

File: README.md [hunk 5]
```diff
@@ -94,6 +94,40 @@ This is the planned roadmap for Cascadia Code. Please be aware that the delivery
...
+
+![Visual Studio 2019 Settings](images/vs2019-font-settings.png "Visual Studio 2019 Font Settings")  
+
+# Installing Cascadia Code in Windows Terminal (Preview)
+
+1. Go to the Dropdown`Preferences` or hit `Ctrl + ,` in Windows Terminal (Preview).
```
Comment (ItzLevvie) [inline README.md]:
```suggestion
1. Go to the Dropdown `Preferences` or hit `Ctrl + ,` in Windows Terminal (Preview).
```

File: README.md [hunk 6]
```diff
@@ -94,6 +94,40 @@ This is the planned roadmap for Cascadia Code. Please be aware that the delivery
...
+2. Open the `profiles.json` in an editor like VS Code.
+3. Scroll down to the `"profiles"` Property.
+4. Look for your desired profile.
+5. change the `"fontFace"` attribute to `"fontFace": "Cascadia Code"`.
+
+![Windows Terminal (Preview) Settings](images/windows-terminal-preview-font-settings.png "Windows Terminal (Preview) Font Settings")
+
+![Windows Terminal (Preview) Settings](images/windows-terminal-preview-font-settings-json.png "Windows Terminal (Preview) Font Settings")
+
+# Installing Cascadia Code in Intellij IDE 2019
```
Comment (ItzLevvie) [inline README.md]:
```suggestion
# Installing Cascadia Code in IntelliJ IDE 2019
```

File: README.md [hunk 7]
```diff
@@ -94,6 +94,40 @@ This is the planned roadmap for Cascadia Code. Please be aware that the delivery
...
+
+1. Go to the Dropdown`File > Settings` or hit `Ctrl + Alt + S` in Intellij IDE 2019.
```
Comment (ItzLevvie) [inline README.md]:
```suggestion
1. Go to the Dropdown `File > Settings` or hit `Ctrl + Alt + S` in IntelliJ IDE 2019.
```

File: README.md [hunk 8]
```diff
@@ -94,6 +94,40 @@ This is the planned roadmap for Cascadia Code. Please be aware that the delivery
...
+2. Enter "Appearance" in search field or go to `Editor > Font`.
+4. Select `Cascadia Code` in the Dropdown.
+5. Select Enable Font Ligatures
+6. Press Ok and you're good to go.
+
+![Intellij IDE 2019 Settings](images/intellij-ide-2019-font-settings.png "Intellij IDE 2019 Font Settings")
```
Comment (ItzLevvie) [inline README.md]:
```suggestion
![IntelliJ IDE 2019 Settings](images/intellij-ide-2019-font-settings.png "IntelliJ IDE 2019 Font Settings")
```

File: README.md [hunk 9]
```diff
@@ -94,6 +94,40 @@ This is the planned roadmap for Cascadia Code. Please be aware that the delivery
...
+
+![Visual Studio 2019 Settings](images/vs2019-font-settings.png "Visual Studio 2019 Font Settings")  
+
+# Installing Cascadia Code in Windows Terminal (Preview)
+
+1. Go to the Dropdown`Preferences` or hit `Ctrl + ,` in Windows Terminal (Preview).
+2. Open the `profiles.json` in an editor like VS Code.
+3. Scroll down to the `"profiles"` Property.
+4. Look for your desired profile.
+5. change the `"fontFace"` attribute to `"fontFace": "Cascadia Code"`.
```
Comment (ItzLevvie) [inline README.md]:
```suggestion
5. Change the `"fontFace"` attribute to `"fontFace": "Cascadia Code"`.
```

**Comment by tklecka** [comment]
Thanks to both of you for checking my text, @cinnamon-msft and @ItzLevvie 👌😜
Any other suggestions?

File: README.md [hunk 10]
```diff
@@ -94,6 +94,40 @@ This is the planned roadmap for Cascadia Code. Please be aware that the delivery
...
+2. Open the `profiles.json` in an editor like VS Code.
+3. Scroll down to the `"profiles"` Property.
+4. Look for your desired profile.
+5. Change the `"fontFace"` attribute to `"fontFace": "Cascadia Code"`.
+
+![Windows Terminal (Preview) Settings](images/windows-terminal-preview-font-settings.png "Windows Terminal (Preview) Font Settings")
+
+![Windows Terminal (Preview) Settings](images/windows-terminal-preview-font-settings-json.png "Windows Terminal (Preview) Font Settings")
+
+# Installing Cascadia Code in IntelliJ IDE 2019
```
Comment (cinnamon-msft) [inline README.md]:
```suggestion
# Setting Cascadia Code in IntelliJ IDE 2019
```

**Comment by cinnamon-msft** [comment]
I just have one last one, then I think it'll be good to merge!

**Comment by tklecka** [comment]
> I just have one last one, then I think it'll be good to merge!
@cinnamon-msft sounds good to me

**Pull request merged**